examples/swift/SwiftWrapperApp/SwiftWrapperApp/ContentView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ContentView.swift
+//  SwiftWrapperApp
+//
+//  Created by Abhinav Saroj on 6/22/23.
+//
+
+import OneDSSwift
+import SwiftUI
+
+struct ContentView: View {
+	@State private var enteredText = ""
+	
+    var body: some View {
+		VStack {
+			TextField("Enter text", text: $enteredText)
+				.textFieldStyle(RoundedBorderTextFieldStyle())
+				.padding()
+
+			Button(action: {
+				// Perform an action with the captured text
+				print("Entered text: \(enteredText)")
+				sendTelemetry(enteredText)
+				print("Telemetry Sent.")
+			}) {
+				Text("Send Telemtery")
+					.padding()
+					.foregroundColor(.white)
+					.background(Color.accentColor)
+					.cornerRadius(10)
+			}
+		}
+		.padding()
+	}
+	
+	/// Method accessing Swift Wrapper package OneDSSwift telemetry classes
+	func sendTelemetry(_ text:String) {
+		let event:EventProperties = EventProperties(name: "SetProps_Swift_Wrapper_Example_Proj")
+		event.setProperty("enteredtext", withValue: text)
+		event.setProperty("from", withValue: "Swift_Wrappers_Package_Example_Project")
+		
+		let logger: Logger? = LogManager.loggerForSource(source: "exmaple_proj_source")
```

Comment:
1) You need to initialize, with a tenant token
2) Typo on source parameter

Reply:
1. I planned to just show that classes/types from swift wrapper are being imported and linked properly, rather than sending actual telemetry events (something that we are doing in the standalone file in main.swift under swift wrappers).

2. fixed the typo